### PR TITLE
configure deployment on playground with circleCI

### PR DESCRIPTION
**Pull request by emmanuelle**

### CODE REVIEW

File: predeploy.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import shutil
+import subprocess
+
+# Install dash-slicer from dev version
+subprocess.run('python -m pip install -e .'.split(' '))
```
Comment (almarklein) [inline predeploy.py]: This should already install the requirements, I think.

Comment (almarklein) [reply predeploy.py]: Oh, nevermind, the below installs the requirements for that specific app.

Comment (emmanuelle) [reply predeploy.py]: I need to install more packages such as `gunicorn` or `imageio` on the remote machine, this is why I added another `requirement.txt`.

Comment (emmanuelle) [reply predeploy.py]: I expect a bit of debugging ahead so please be patient :-).

**Comment by emmanuelle** [comment]
Still WIP since the creation of new apps from the CI does not work yet, but apart from this the deployment works.

**Comment by emmanuelle** [comment]
@almarklein I think this is now ready for review (I'll squash commits before merging :-).

File: README.md [hunk 2]
```diff
@@ -45,3 +45,7 @@ This code is distributed under MIT license.
 * Use `black .` to autoformat.
 * Use `flake8 .` to lint.
 * Use `pytest .` to run the tests.
+
+On every PR, an app with the same name as your branch is deployed to the Dash
+playground instance so that you can change whether your changes did not break
```
Comment (almarklein) [inline README.md:50]: typo: "change" -> "check"

File: test_deploy/app.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+"""
+An example with two slicers on the same volume.
+"""
```
Comment (almarklein) [inline test_deploy/app.py:3]: Would be interesting to eventually put an example app here, that covers most of the slicer's functionality.

**Comment by almarklein** [comment]
Out of curiosity, I assume the playground is used by more dash-related projects. If two projects that use it happen to have a pr open with a branch name that is the same, we'll have a name clash, right? (not a big deal, but good of be aware of)

**Comment by almarklein** [comment]
This LGTM, by the way!

**Comment by emmanuelle** [comment]
> Out of curiosity, I assume the playground is used by more dash-related projects. If two projects that use it happen to have a pr open with a branch name that is the same, we'll have a name clash, right? (not a big deal, but good of be aware of)

You're right and I considered that this is not a big deal since the playground is just a sandbox (used for things like continuous deployment and work in progress). But it's easy to prefix the branch name with 'slicer' so that the risk of fortuitous clashes is very much reduced, I'll do that.

**Comment by almarklein** [comment]
I'll avoid branch names called "dev" from now on ;)

**Pull request merged**